src/Sarif/Readers/LineMappingStreamReader.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.CodeAnalysis.Sarif.Readers
     ///  in the part of the file currently read. This allows it to map a line and char back to a byte offset
     ///  to allow consumers to seek back to the position later.
     /// </summary>
-    internal class LineMappingStreamReader : StreamReader
```

Comment:
>internal [](start = 4, length = 8)

Why this change?

Reply:
Left from running testing in another project because I couldn't get the SDK unit tests to execute. Undone.